### ISSUE TITLE
Deprecation Warning

### ISSUE THREAD

**Issue by alinvetian** [mperham/sidekiq#5344]
Hello,

I am using sidekiq-ent 2.2.3 and I get the bellow warning.
Should this be fixed in version 2.3.1?
Also why can't I install a newer version than 2.2.3?

Here is my environment:
```
2022-05-26T08:01:03.034Z pid=69061 tid=1j2l INFO: Booted Rails 6.0.4 application in development environment
2022-05-26T08:01:03.034Z pid=69061 tid=1j2l INFO: Running in ruby 2.7.2p137 (2020-10-01 revision 5445e04352) [x86_64-linux]
2022-05-26T08:01:03.035Z pid=69061 tid=1j2l INFO: Sidekiq Pro 5.2.4 / Sidekiq Enterprise 2.2.3, commercially licensed.
2022-05-26T08:01:03.035Z pid=69061 tid=1j2l INFO: Booting Sidekiq 6.4.2 with redis options {}
2022-05-26T08:01:03.036Z pid=69061 tid=1j2l INFO: Starting processing, hit Ctrl-C to stop
2022-05-26T08:01:03.040Z pid=69061 tid=123h INFO: Managing 27 periodic jobs
Pipelining commands on a Redis instance is deprecated and will be removed in Redis 5.0.0.

redis.pipelined do
  redis.get("key")
end

should be replaced by

redis.pipelined do |pipeline|
  pipeline.get("key")
end

(called from ~/.rvm/gems/ruby-2.7.2@justknock/gems/sidekiq-ent-2.2.3/lib/sidekiq-ent/periodic/config.rb:67:in `block in persist'}
2022-05-26T08:01:03.042Z pid=69061 tid=123h INFO: Gained leadership of the cluster
Pipelining commands on a Redis instance is deprecated and will be removed in Redis 5.0.0.

redis.pipelined do
  redis.get("key")
end

should be replaced by

redis.pipelined do |pipeline|
  pipeline.get("key")
end

(called from ~/.rvm/gems/ruby-2.7.2@justknock/gems/sidekiq-ent-2.2.3/lib/sidekiq-ent/periodic/manager.rb:179:in `block in enqueue_job'}
```

**Comment by mperham** [comment]
> Should this be fixed in version 2.3.1?

It is fixed.

> Also why can't I install a newer version than 2.2.3?

I don't know. Are you using the latest Bundler? Do you have version restrictions in your Gemfile.lock?

**Comment by alinvetian** [comment]
Thank you @mperham for your answer. I seem that the issue was a locked bundler version on my side.
All looks good now.

The issue can be closed.
